_sass/_base.scss

```diff
@@ -4,7 +4,7 @@
 body, h1, h2, h3, h4, h5, h6,
 p, blockquote, pre, hr,
 dl, dd, ol, ul, figure {
-  margin: 0;
+  //margin: 0;
```

Comment:
I thought that I accidentally had commented this out, which is why I put it
back in. Just switched it in the last branch and I didn't notice any change
with my designs. =)

On Sat, Feb 28, 2015 at 4:32 PM, Mario Olivio Flores <
notifications@github.com> wrote:

> In _sass/_base.scss
> <https://github.com/MayOneUS/mayday-2.0-frontend/pull/15#discussion_r25561448>
> :
>
> > @@ -4,7 +4,7 @@
> >  body, h1, h2, h3, h4, h5, h6,
> >  p, blockquote, pre, hr,
> >  dl, dd, ol, ul, figure {
> > -  margin: 0;
> > +  //margin: 0;
>
> This is a pretty big global change and usualy a standard reset/normalize
> move. I'd actually prefer to keep this. Not sure why we would comment it
> out. Is there some backstory I'm missing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MayOneUS/mayday-2.0-frontend/pull/15/files#r25561448>.
>